frontend/packages/console-app/src/components/namespace/filters.ts

```diff
@@ -0,0 +1,39 @@
+import { SYSTEM_NAMESPACES_PREFIX, SYSTEM_NAMESPACES } from '@console/shared';
+// unsure why getActiveUserName is triggering a no-cycle error.
+// eslint-disable-next-line import/no-cycle
+import { getActiveUserName } from '../../../../../public/actions/ui';
+
+export const isCurrentUser = (user: string): boolean => user === getActiveUserName();
+
+export const isSystemNamespace = (option: { title: string; key?: string }) => {
+  const startwithNamespace = SYSTEM_NAMESPACES_PREFIX.some((ns) => option.title?.startsWith(ns));
+  const isNamespace = SYSTEM_NAMESPACES.includes(option.title);
+
+  return startwithNamespace || isNamespace;
+};
+
+export const isOtherUser = (user: string, title: string): boolean => {
+  return !isCurrentUser(user) && !isSystemNamespace({ title });
+};
+
+export const requestorFilter = (filter, obj): boolean => {
```

Comment:
nit, this and other instances of `requestor` should match spelling of the underlying API
```suggestion
export const requesterFilter = (filter, obj): boolean => {
```

Reply:
Fixed references to request**er**